components/brave_wallet_ui/components/shared/buy-option/buy-asset-option.styles.ts

```diff
@@ -5,9 +5,13 @@
 
 import styled from 'styled-components'
 import * as leo from '@brave/leo/tokens/css'
-import { WalletButton } from '../style'
+import { WalletButton, Row } from '../style'
 
-export const BuyAssetOptionWrapper = styled(WalletButton)<{
+export const ButtonWrapper = styled(Row)`
+  padding: 6px 12px;
+`
```

Comment:
Looks like we could render a `Row` with a set padding prop instead

Reply:
Good catch, fixed!